app/assets/stylesheets/mobile/components/user-card.scss

```diff
@@ -23,14 +23,15 @@ $avatar_width: 120px;
       flex-wrap: wrap;
       margin: 1em calc(var(--usercard-control-margin) * -1) 0;
       li {
-        flex: 1 0 auto;
+        flex: 1 0 calc(50% - (var(--usercard-control-margin) * 2));
```

Comment:
Can you explain what `calc(50% - (var(--usercard-control-margin) * 2))` does?

Reply:
Sure,

It allows each button to take up half of the available space. It is subtracting the margin from right and left of the button to account that it doesn't go over 50% of that space, with margins included.

Reply:
Thanks. So if there are three buttons, buttons 1 and 2 will take up 50% each, but will the third button also take up half the available space?

Reply:
It will take up the full amount, since its `flex-grow` is set to `1` with `flex: 1 0 ...`

The third number here is just the *base*, from where it will then obey the `flex-grow` or `flex-shrink` settings.